server/integration-tests/listener.html

```diff
@@ -5,18 +5,41 @@
 <style>
   body { font-family: 'Helvetica', 'Arial', sans-serif; }
 </style>
-<p>
+<fieldset>
+  <legend>Configuration</legend>
+  URL: <input id="url" type="url" value="ws://localhost:3000"/>
+  <button id="reload" type="button">Reload</button>
+</fieldset>
+<fieldset>
+  <legend>Status</legend>
   <!-- we set the status dynamically from the websocket data -->
-  Status: <span id="status"></span>
-</p>
+  <pre id="status"></pre>
+</fieldset>
```

Comment:
HTML really changed since 1999.

Reply:
`fieldset` and `pre` were added in December 1999 😄